charts/kubernetes-agent/README.md

```diff
@@ -36,6 +36,7 @@ A Helm chart for the Octopus Kubernetes Agent
 | agent.resources | object | `{"requests":{"cpu":"100m","memory":"150Mi"}}` | The resource limits and requests assigned to the agent container |
 | agent.serverApiKey | string | `""` | An Octopus Server API key use to authenticate with the target Octopus Server |
 | agent.serverCommsAddress | string | `""` | The polling communication URL of the target Octopus Server |
+| agent.serverCommsAddresses | list | `[]` | The polling communication URLs of the target Octopus Servers when running in HA |
```

Comment:
```suggestion
| agent.serverCommsAddresses | list | `[]` | The polling communication URL's of the target Octopus Servers when running in High Availability (HA) |
```

Reply:
I don't think an apostrophe is right on "URL's" because it's plural, not possessive in this context.

Reply:
ah yep